controllers/nim_account_controller.go

```diff
@@ -266,7 +266,7 @@ func (r *NimAccountReconciler) reconcileNimConfig(
 		return nil, dErr
 	}
 
-	cmCfg := ssacorev1.ConfigMap(constants.NimDataConfigMapName, namespace).
```

Comment:
Since it is no longer static, would it make sense to add some assertions in the units?

Reply:
Added in https://github.com/opendatahub-io/odh-model-controller/pull/340/commits/25027d88b599aa8fb9876334b8b570cfe6775c53